bundles/net.tourbook/src/net/tourbook/tour/filter/SlideoutTourFilter.java

```diff
@@ -795,12 +796,7 @@ private void createUI_430_FieldDetail(final TourFilterProperty filterProperty) {
       final TourFilterFieldType fieldType = fieldConfig.fieldType;
 
       switch (filterProperty.fieldOperator) {
-      case GREATER_THAN:
-      case GREATER_THAN_OR_EQUAL:
-      case LESS_THAN:
-      case LESS_THAN_OR_EQUAL:
-      case EQUALS:
-      case NOT_EQUALS:
+      case GREATER_THAN,GREATER_THAN_OR_EQUAL, LESS_THAN, LESS_THAN_OR_EQUAL, EQUALS, NOT_EQUALS:

```

Comment:
Reverted.

Can you please deactivate it from SonarCloud by going [here](https://sonarcloud.io/organizations/wolfgang-ch/rules?open=java%3AS6208&rule_key=java%3AS6208)? That way, SonarCloud won't flag it again

I should have this button "Deactivate" but I don't think I have the permissions since it's your codebase

![image](https://github.com/mytourbook/mytourbook/assets/25235665/398daa53-20e6-4742-a959-3b2976193f18)

Reply:
Done 

![image](https://github.com/mytourbook/mytourbook/assets/1283445/fab3b5f5-9a19-4a37-96ac-eff671393b96)